native/packages/react-native-bpk-component-button-link/src/BpkButtonLink.ios.js

```diff
@@ -77,37 +79,36 @@ const BpkButtonLink = (props: Props) => {
   }
 
   if (disabled) {
-    textStyle.push(styles.disabled);
+    textStyle.push(styles.textDisabled);
     accessibilityTraits.push('disabled');
   }
 
-  if (style) {
-    containerStyle.push(style);
-  }
-
   return (
-    <View style={containerStyle}>
-      <TouchableOpacity
-        accessibilityComponentType="button"
-        accessibilityLabel={accessibilityLabel || title}
-        accessibilityTraits={accessibilityTraits}
-        onPress={onPress}
-        disabled={disabled}
-        icon={icon}
-        {...rest}
-      >
-        <View style={viewStyle}>
-          <BpkText textStyle={large ? 'lg' : 'sm'} emphasize style={textStyle}>
-            {title}
-          </BpkText>
-          {typeof icon === 'string' ? (
-            <BpkIcon icon={icon} style={iconStyle} small={!large} />
-          ) : (
-            icon
-          )}
-        </View>
-      </TouchableOpacity>
-    </View>
+    <TouchableOpacity
```

Comment:
I've removed the outer view here - `TouchableOpacity` already wraps `children` in a `<View />`.